tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_participants_under_18years.py

```diff
@@ -0,0 +1,150 @@
+"""
+Integration test for remove_participants_under_18years module
+
+Original Issues: DC-1724
+
+The intent is to remove data for participants under 18 years old  from all the domain tables."""
+
+# Python Imports
+import os
+import datetime
+
+# Project Imports
+from common import VISIT_OCCURRENCE, OBSERVATION
+from common import JINJA_ENV
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.remove_participants_under_18years import (
+    RemoveParticipantsUnder18Years, AFFECTED_TABLES)
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import \
+    BaseTest
+
+PERSON_DATA_TEMPLATE = JINJA_ENV.from_string("""
+INSERT INTO `{{project_id}}.{{dataset_id}}.person`
+(person_id, birth_datetime, gender_concept_id, year_of_birth, race_concept_id, ethnicity_concept_id)
+VALUES
+      (1, '1970-01-01 00:00:00 EST', 0, 1970, 0, 0),
+      (2, '2002-01-01 00:00:00 EST', 0, 2002, 0, 0),
+      (3, '2003-01-01 00:00:00 EST', 0, 2003, 0, 0),
+      (4, '2015-01-01 00:00:00 EST', 0, 2015, 0, 0)
```

Comment:
NIT: Some poor engineer 12 years from now will be confused why their tests are failing 😁

Reply:
@jp3477 Now that poor engineer can avoid it for 5 more years 😉. TBH I couldn't think of another approach to avoid this.